var/spack/repos/builtin/packages/py-ansimarkup/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAnsimarkup(PythonPackage):
+    """Produce colored terminal text with an xml-like markup."""
+
+    homepage = "https://github.com/gvalkov/python-ansimarkup"
+    pypi = "ansimarkup/ansimarkup-1.5.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("1.5.0", sha256="96c65d75bbed07d3dcbda8dbede8c2252c984f90d0ca07434b88a6bbf345fad3")
+
+    depends_on("python@2.7,3.3:3.9", type=("build", "run"))
```

Comment:
Where do you get the upper restriction from?

Reply:
Thank you for your review. I add the latest version of ansimarkup and remove the dependency on python (I get the upper restriction from the setup.py but @adamjstewart pointed out to me not to take into account the content of classifiers.)